components/validator/challenge/fetcher.go

```diff
@@ -82,3 +81,51 @@ func Decode(data []byte) []*big.Int {
 
 	return result
 }
+
+var _ ProverClient = (*JsonRPCProverClient)(nil)
```

Comment:
Why do we need this line?

Reply:
This is for explicit implementation of interface in go, which means `JsonRPCProverClient` implements `ProverClient` interface.
ref: https://go.dev/doc/effective_go#blank_implements